spec/poros/food_spec.rb

```diff
@@ -0,0 +1,8 @@
+require 'rails_helper'
+
+RSpec.describe Food do 
+    xit 'exists' do 
+        food = Food.new(total_hits: 1, description: "liter of cola", code: "123", brand_owner: "Coca-Cola", ingredients: "crushed dreams")
+        expect(food).to be_an_instance_of(Food)
```

Comment:
Saw your note about brain fart with the # of arguments here -- makes sense! We would just expect to see assertions about the exact attributes in the food object